solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,156 @@
+"""Simulating an ATM. """
+
+import sys
+import json
+
+ATM_FILE = 1  # The number 1 refers to the place of the file as a parameter.
+
+
+def cash_withdrawal(costumer_id, costumers_data):
+    """Withdrawal from the ATM amount of money.
+
+    Args:
+        costumer_id (string): Costumer id.
+        costumers_data (dictionary): Bank costumer data.
+
+    Returns:
+        boolean: True for success withdrawal action, False otherwise.
+    """
+    amount_to_withdrawal = float(input("Amount to withdrawal: "))
+    balance_now = float(costumers_data[costumer_id]['balance'])
+    if 0 < amount_to_withdrawal <= balance_now:
+        new_balance = balance_now - amount_to_withdrawal
+        costumers_data[costumer_id]['balance'] = new_balance
+        return True
+
+    return False
+
+
+def cash_deposit(costumer_id, costumers_data):
+    """Deposit money to the bank.
+
+    Args:
+        costumer_id (string): Costumer id.
+        costumers_data (dictionary): Bank costumer data.
+
+    Returns:
+        boolean: True for success deposit action, False otherwise.
+    """
+    amount_to_deposit = float(input("Amount to deposit: "))
+    if amount_to_deposit > 0:
+        balance_now = float(costumers_data[costumer_id]['balance'])
+        new_balance = balance_now + amount_to_deposit
+        costumers_data[costumer_id]['balance'] = new_balance
+        return True
+
+    return False
+
+
+def change_password(costumer_id, costumers_data, new_password):
+    """Changing costumer's ATM password.
+
+    Args:
+        costumer_id (string): Costumer id.
+        costumers_data (dictionary): Bank costumer data.
+        new_password (string): Password the costumer wants to change to.
+
+    Returns:
+        boolean: True for success changing the password, False otherwise.
+    """
+    if len(new_password) == 4 and new_password.isdigit():
+        costumers_data[costumer_id]['password'] = new_password
+        return new_password
+
+    return False
+
+
+def create_dictionary(path_of_file):
+    """Create dictionary of bank customer data.
+
+    Args:
+        path_of_file (string): Path of the file that contains all bank
+        costumer data.
+
+    Returns:
+        dictionary: A nested dictionary of bank costumer data, or an exception
+        if the file path is not correct.
```

Comment:
if you got a variable \ return value in the doc that require more then a single line, the next line should be with indent (for example line 72 "customer data" should be with indent)